module/apmhttp/client.go

```diff
@@ -83,6 +84,7 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	if tx == nil {
 		return r.r.RoundTrip(req)
 	}
+	ctx = r.requestTracer.start(ctx)
```

Comment:
Instead of creating spans as direct children of the transaction, we should make them children of the `external.http` span below. (And only if StartSpan returns a non-dropped span.)